### ISSUE TITLE
Cannot upload Stripe apps using Stripe CLI

### ISSUE THREAD

**Issue by jwright** [stripe/stripe-apps#730]
**Describe the bug**
When starting a new app upload using `stripe apps upload`, I am presented with the following:

![CleanShot_2022-09-06_at_12 31 562x](https://user-images.githubusercontent.com/35017/188902270-114c6009-beea-4315-96a1-7946425511b5.png)

When I hit enter, I am sent to the browser and I am presented with a success dialog.

<img width="510" alt="CleanShot_2022-09-06_at_12 32 222x" src="https://user-images.githubusercontent.com/35017/188902455-500108a4-c09f-44e9-9d29-a9441aeeca18.png">

However, this prompt never goes away. It is still there when I restart the process as well.

I have tried the following with no success:

1. Log out of web and CLI and try again
2. Upgrade the Stripe CLI
3. Try various browsers

I am on Stripe CLI version 1.11.3.

**To Reproduce**
1. Using the Stripe CLI, use `stripe apps upload`
2. Notice the "Accept Terms"
3. Hit "enter"
4. Notice success banner
5. Go back to CLI
7. See same "Accept Terms" output

**Expected behavior**
Once I accept the new terms and get a success screen, then I should not have to agree with them again and I can upload my app.

**Screenshots**
See above.

**Desktop (please complete the following information):**
 - OS: MacOS Monterey 12.5.1
 - Browser: Tried on Chrome and Safari

**Additional context**
This may just be an issue with my account...or I would assume this would be a larger issue?

**Comment by htheodore-stripe** [comment]
Hi Jamie,

Would you try unsetting your `STRIPE_API_KEY` from your environment variables and try the commands again? Currently Stripe apps CLI doesn't support API keys yet and we've seen similar issues before because API key was being used.

**Comment by jwright** [comment]
@htheodore-stripe Hmmm. I do not have a `STRIPE_API_KEY` environment variable set locally. Nor do I use the `--api-key` option with the stripe apps command.

What am I missing here?

**Comment by htheodore-stripe** [comment]
Would you try upgrading the stripe apps CLI as well and try again? `stripe plugin upgrade apps`

And just to be sure, you have run `stripe login` as well right from the CLI? To make sure you are using the same account on the Stripe Dashboard and the Stripe CLI.

**Comment by jwright** [comment]
@htheodore-stripe Yes! So I was not signed into the correct account. It was weird because I thought I was (but I was not due to my password manager). However, I could still run and trigger webhooks using the account I thought I was logged in under, which is why I always thought I was logged in under that account.

A `stripe whoami` command may go a long way for support :)

Thank you for the reply!
